core/src/AbilityHelpers.ts

```diff
@@ -264,3 +271,21 @@ export function getAbilityHelpersAttribute(
         [Types.AbilityHelpersAttributeName]: attr
     };
 }
+export function abilityHelpersExists(win: Window): boolean {
```

Comment:
Probably instead of two methods (abilityHelpersExists and getWindowInstance) makes sense to do just getCurrentAbilityHelpers(win).

Reply:
Yeah that's a good call, done

Reply:
Also log a warning and return an existing instance if it exists, error could be a bit too far